main.cpp

```diff
@@ -722,15 +722,15 @@ int main(int argc, char *argv[])
 {
     Mainloop &mainloop = Mainloop::init();
 
-    log_open();
+    Log::open();
```

Comment:
Do we need to have open/close functions? Can't we use static initializes for that? Or a singleton design pattern?